deploy-as-code/helm/charts/health-services/project-factory/values.yaml

```diff
@@ -252,6 +252,8 @@ env: |
     value: {{ .Values.ExcludeHierarchyTypeFromBoundaryCodes | default "false" | quote }}
   - name: EXCLUDE_BOUNDARY_NAME_AT_LAST_FROM_BOUNDARY_CODES
     value: {{ .Values.ExcludeBoundaryTypeFromLastBoundaryCodes | default "false" | quote }}
+  - name: SPRING_CODEC_MAX_IN_MEMORY_SIZE
```

Comment:
this property has to be added in gateway

Reply:
ok